lib/postgres.js

```diff
@@ -249,6 +251,26 @@ class Postgres {
                 }
             }
 
+            const keychains = item.keychains.length > 0 ? item.keychains.map((s) => {
+                const res = {s: s.slot, i: s.sticker_id};
+                if (s.pattern) {
+                    res.pattern = s.pattern;
+                }
+                if (s.name) {
```

Comment:
nit: You should probably save all the field (even if CS2 isn't populating them yet) since it uses the same sticker proto.